src/mudlet-lua/lua/generic-mapper/generic_mapper.xml

```diff
@@ -4159,6 +4159,7 @@ function map.eventHandler(event, ...)
         end
     elseif event == "sysLoadEvent" or event == "sysInstall" then
         config()
+        raiseEvent("mapperScriptLoaded", "generic_mapper")
```

Comment:
Consider https://wiki.mudlet.org/w/Manual:Event_Engine#Mudlet-raised_events - would a rename of the event to `sysMapperPackageLoaded` be a suitable alternative name for the event (and the similar one I believe you are intending for the IRE one)?